public/js/views/management/pay-methods.jsx

```diff
@@ -35,6 +40,12 @@ export default class PayMethods extends Component {
   }
 
   render() {
+
+    var isDeleteDisabled = !this.props.payMethods ||
+      !this.props.payMethods.length;
+    var deleteClasses = cx('button', 'quiet', 'delete',
+                           {'disabled': isDeleteDisabled});
```

Comment:
I think this would be less confusing as `isDisabled` but I see what you were doing.